config/default.yaml

```diff
@@ -32,7 +41,7 @@ modules:
 
   collectors:
   # Uncomment the modules that you'd like to use
-    # - ./modules/collectionLogger
+    - ./modules/collectionLogger
```

Comment:
In the last PR I sent you I commented them all out, I'd have to double check but I believe I ran into an error if no collectors were available. I also thought it would be best to have the logger for the initial setup so that users can test that the endpoint is being reached before adding database config. I can comment this out again if you'd like.